src/test/koa-module-specifier-transform.test.ts

```diff
@@ -18,6 +18,49 @@ import {moduleSpecifierTransform} from '../koa-module-specifier-transform';
 
 import {createAndServe, squeeze, testLogger} from './test-utils';
 
+test(
+    'moduleSpecifierTransform default parse/serialize preserves tags',
+    async (t) => {
+      t.plan(1);
+      const logger = testLogger();
+      createAndServe(
+          {
+            middleware:
+                [moduleSpecifierTransform((_, __, ___) => './x.js', {logger})],
+            routes: {
+              '/my-page.html': `
+                <html>
```

Comment:
maybe add `<!DOCTYPE html>` just to make sure that sticks around too?

Reply:
ok!